librz/include/rz_rop.h

```diff
@@ -22,13 +22,16 @@ extern "C" {
  */
 typedef struct rz_rop_reg_info_t {
 	char *name;
+	bool is_cast;
```

Comment:
Remove this please. It is not used (at least according to my quick search).

Reply:
Yes, I will be adding this later, removed it for now